src/pybind/mgr/dashboard/tests/__init__.py

```diff
@@ -124,7 +124,9 @@ def setup_controllers(cls, ctrl_classes, base_url=''):
         cherrypy.tree.mount(None, config={
             base_url: {'request.dispatch': mapper}})
 
-    def __init__(self, *args, **kwargs):
+    @classmethod
+    def setup_class(cls):
```

Comment:
Mmm, as this is `unittest` (even if run by `pytest`), why not using [`setUpClass()`](https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.setUpClass) instead? Same for `teardown_class` vs. `tearDownClass`.

Reply:
fixed.

Reply:
fixed.